packages/nimble-components/src/tabs/specs/scrollable-tabs.md

```diff
@@ -23,6 +23,8 @@ The following decisions were agreed to in the comments section in the linked iss
 -   Keyboard navigation will work as it does today, but when a tab that isn't fully in view gets focus it should be scrolled into view
 -   When a scroll button is pressed we will scroll either a full "page width" or the remainder of the available space, and tab focus is not changed.
 
+Additionally, devices like mouse wheels and trackpads as well as pan gestures on mobile devices should allow scrolling through the tabs. This should be handled natively by the browser by simply applying CSS of `overflow: scroll;` to the div containing the tabs. _Note: Mouse users would have to hold `<Shift>` while using the scroll wheel in order to scroll the tabs. This is consistent with other horizontally scrollable content._
```

Comment:
> This is consistent with other horizontally scrollable content

I haven't done any kind of exhaustive survey, but consider the following cases where tab scrolling or focus is moved horizontally without `Shift`:
1. VS Code tabs
2. Windows Explorer tabs
3. Notepad++ tabs
4. Outlook ribbon hubs (e.g. File, Home, etc)

When there is only one scrollable axis (vertical or horizontal), it should scroll using just the mouse wheel. Only when there are two scrollable dimensions (like in a Figma doc or Excel spreadsheet) does it make sense to use `Shift` to scroll the secondary dimension. Even then, it's not completely consistent, e.g. Excel uses `Ctrl`+`Shift`+wheel for horizontal scrolling.

Reply:
I'm certainly not against this decision. I was just using the browser as an example when there was only a horizontal scrollbar, it required the `<Shift>` key, but this could reasonably be simply because there is opportunity for there to also be vertically scrollable content, and they want to have consistent expectations for any keyboard modifiers. In this case, there is no such opportunity, so I agree, it would be nice for the user to allow it without using `<Shift>`. Thoughts @jattasNI?

Reply:
My goals are:
1. match the user expectations for the platform. 
2. avoid unnecessary work and implementation complexity if we can.

For 1, I'm iffy on Mert's examples because they are Windows apps not websites (granted some of them may be implemented in HTML). My suspicion is that browser users wouldn't expect this behavior, but I could be proven wrong with more examples from the web. What does the scrollwheel on Windows in a browser on horizontally scrollable content that isn't embedded in a vertically scrollable page? To test, you could open the codepen for the [shoelace scrollable tabs example](https://shoelace.style/components/tab-group#scrolling-tabs).

For 2, my understanding is that this would add an outsize amount of complexity because we'd have to handle scroll events and translate them to offsets manually versus just letting the browser handle the scroll. That opens the door to the scrolling not feeling right and would necessitate more code and test cases. But I could be wrong about that.

Reply:
To answer your question about the shoelace scrollable tabs behavior on Windows, it does, indeed, need `<Shift>` to scroll.

Reply:
> _Note: Mouse users would have to hold `<Shift>`

This meets my expectations - 👍